keyboards/viktus/sp111/sp111.h

```diff
@@ -23,14 +23,14 @@
 	L10, L11, L12, L13,   L14, L15, L16, L17, L18, L19, L0B,   	R10, R11, R12, R13, R14, R15, R16, R17,	   R18, R19, \
```

Comment:
tabs for indent/alignment :scream:

Reply:
Don't worry, i hate myself for doing it too. Lets fix that up!